test/test_worker.py

```diff
@@ -20,9 +20,12 @@ def setUp(self):
 
     def test_proctitle(self):
         '''Make sure we can get / set the process title'''
-        before = Worker.title()
-        Worker.title('Foo')
-        self.assertNotEqual(before, Worker.title())
+        try:
```

Comment:
```python
try:
    import setproctitle
    before = Worker.title()
    ...
except ImportError:
    pass
```

That way we only run this test if the environment actually has `setproctitle` available to it. Then we can re-enable the `except ImportError` block in `__init__`